tools/ua-dev-cloud-config.yaml

```
@@ -9,4 +9,4 @@ runcmd:
  - make deps
  - dpkg-buildpackage -us -uc
  - apt-get remove ubuntu-advantage-tools
- - dpkg -i /var/tmp/ubuntu-advantage-tools_20.4_amd64.deb
+ - dpkg -i /var/tmp/ubuntu-advantage-tools_25.0_amd64.deb
```

Let's avoid pinning the dev config to a specific version of ua-t and use globbing '*' instead so we don't have to increment this in lockstep with each new version, *and* we can install ubuntu-advantage-pro*deb too

Also, I think we need an --assume-yes on the apt-get remove in case there are depedencies that will also get uninstalled (on groovy)

```suggestion
- apt-get remove ubuntu-advantage-tools --assume-yes
- dpkg -i /var/tmp/ubuntu-advantage-*deb
```